pkg/gui/controllers/branches_controller.go

```diff
@@ -297,35 +298,94 @@ func (self *BranchesController) delete(branch *models.Branch) error {
 	if checkedOutBranch.Name == branch.Name {
 		return self.c.ErrorMsg(self.c.Tr.CantDeleteCheckOutBranch)
 	}
-	return self.deleteWithForce(branch, false)
-}
 
-func (self *BranchesController) deleteWithForce(selectedBranch *models.Branch, force bool) error {
-	title := self.c.Tr.DeleteBranch
-	var templateStr string
-	if force {
-		templateStr = self.c.Tr.ForceDeleteBranchMessage
-	} else {
-		templateStr = self.c.Tr.DeleteBranchMessage
+	menuTitle := utils.ResolvePlaceholderString(
+		self.c.Tr.DeleteBranchTitle,
+		map[string]string{
+			"selectedBranchName": branch.Name,
+		},
+	)
+
+	menuItems := []*types.MenuItem{
+		{
+			Label: self.c.Tr.DeleteLocalBranch,
+			Key:   'l',
+			OnPress: func() error {
+				self.c.LogAction(self.c.Tr.Actions.DeleteLocalBranch)
+				return self.localDelete(branch)
+			},
+		},
+		{
+			Label: self.c.Tr.DeleteRemoteBranch,
+			Key:   'r',
+			OnPress: func() error {
+				self.c.LogAction(self.c.Tr.Actions.DeleteRemoteBranch)
+				return self.remoteDelete(branch, false)
+			},
+		},
+		{
+			Label: self.c.Tr.DeleteBranch,
+			Key:   'd',
```

Comment:
Can we change this to 'b' for 'both'? Reason being: I've had people complain about accidentally double-pressing a key and it doing damage and deleting a remote branch may indeed do some damage!

Reply:
I'll resolve this conversation because we decided to remove the `both` option (I'm trying to clean this conversation to keep in mind what I really need to fix). But thanks for pointing this out; I couldn't think about this!